src/index.js

```diff
@@ -1,45 +1,69 @@
 import './style.css';
+import {
+  todos, addTask, deleteTask, editTask, loadTasks,
+} from './todo.js';
 
-const todos = [
-  {
-    index: 1,
-    description: 'Clean My Room',
-    completed: true,
-  },
-  {
-    index: 2,
-    description: 'Do homework',
-    completed: false,
-  },
-  {
-    index: 3,
-    description: 'Buy groceries',
-    completed: false,
-  },
-  {
-    index: 4,
-    description: 'pray',
-    completed: false,
-  },
-  {
-    index: 5,
-    description: 'Go for a run',
-    completed: false,
-  },
-];
+// console.log(todos);
 
 function showList() {
+  loadTasks();
+
   const taskList = document.getElementById('todo-list');
+
   taskList.innerHTML = '';
+
   todos.sort((a, b) => a.index - b.index);
 
-  todos.forEach((todo) => {
+  todos.forEach((todo, index) => {
     const item = document.createElement('li');
-    item.textContent = todo.description;
-    if (todo.completed) {
-      item.classList.add('completed');
-    }
+    const itemDiv = document.createElement('div');
+    const itemSpan = document.createElement('span');
```

Comment:
- [ ] Please use `input` field with type ` checkbox` for your todo list items instead of `span` as you need to mark the list item as completed as well and you can't to do this with plain text in span block.